src/annotator/config/test/url-from-link-tag-test.js

```diff
@@ -0,0 +1,67 @@
+import { urlFromLinkTag } from '../url-from-link-tag';
+
+describe('annotator/config/url-from-link-tag', () => {
+  function appendLink(href, rel, type) {
+    const link = document.createElement('link');
+    link.type = 'application/annotator+' + type;
+    link.rel = rel;
+    if (href) {
+      link.href = href;
+    }
+    document.head.appendChild(link);
+    return link;
+  }
+
+  it('returns the url from the matching link tag of type html', () => {
+    const link = appendLink('http://example.com/app.html', 'sidebar', 'html');
+    assert.equal(
+      urlFromLinkTag(window, 'sidebar', 'html'),
+      'http://example.com/app.html'
+    );
+    link.remove();
+  });
+
+  it('returns the url from the matching link tag of type javascript', () => {
+    const link = appendLink(
+      'http://example.com/app.html',
+      'hypothesis-client',
+      'javascript'
+    );
+    assert.equal(
+      urlFromLinkTag(window, 'hypothesis-client', 'javascript'),
+      'http://example.com/app.html'
+    );
+    link.remove();
+  });
+
+  context('when there are multiple annotator+html notebook links', () => {
```

Comment:
Given that we're testing a generic utility here, I would suggest that the tests also need to be described in a similarly generic way, though they may use parameters/data from a real use case. In this case, the `context` block's description could be "when there are multiple matching links" and then the inner `it` test's description would be "returns the href from the first matching link".